implementations/rust/ockam/ockam_command/src/enroll/mod.rs

```diff
@@ -375,7 +371,7 @@ impl Auth0Service {
 
         opts.terminal
             .write_line(&fmt_log!(
-                "To enroll we need to associate your Ockam identifier with an Orchestrator account\n"
+                " To enroll we need to associate your Ockam identity with an Orchestrator account:\n"
```

Comment:
Is the space at the beginning inentional? I think there is also another weird space at the end of line 66 before the `...`.